client/components/public/FAQ.jsx

```diff
@@ -184,15 +234,22 @@ FAQ = class FAQ extends Component {
             <br /><br />
             We recommend 4-6 people on a team. It can be an advantage to divvy up the work (cutting, constructing, googling, etc.).
           </Accordion.Content>
-
+          <Accordion.Title active={activeIndex === 17} index={17} onClick={(e,p) => this.handleClick(e,p)} >
```

Comment:
This is the goofy part about this FAQ page setup. Is there any trouble with the index being 17 even though that's out of order? I would hope not but not sure

Reply:
We have one other FAQ item that is out of order (16 I think), I haven't noticed any problems with it! I don't think the order of the items actually matters (maybe index isn't the best name for the identifier?)

Reply:
okay, awesome, thanks